docs/esp8266/tutorial/intro.rst

```diff
@@ -99,16 +99,16 @@ And then deploy the new firmware using::
 
     esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect 0 esp8266-20170108-v1.8.7.bin
 
-You might need to change the "port" setting to something else relevant for your
-PC.  You may also need to reduce the baudrate if you get errors when flashing
-(eg down to 115200).  The filename of the firmware should also match the file
-that you have.
+You must use the `port <https://github.com/espressif/esptool#serial-port>`_
+setting for your operating system. You may also need to reduce the baudrate if
```

Comment:
Perhaps 'use the appropriate serial port configuration for your operating system (see <link>)' ?

Reply:
Agree, "You must" is redundant but given link functionality is well understood so is "(see)".